test/e2e/framework/ingress_utils.go

```diff
@@ -761,9 +761,20 @@ func (cont *GCEIngressController) deleteStaticIPs() error {
 func gcloudComputeResourceList(resource, regex, project string, out interface{}) {
 	// gcloud prints a message to stderr if it has an available update
 	// so we only look at stdout.
+	var regexFlag string
```

Comment:
If an additional commit is the standard way, I'll go with that.

Update pushed, I realised `filterFlag` is a better name for the flag given it's the flag applying the filter and not always a regex.